src/Symfony/Component/Security/Core/Tests/Authentication/Token/AbstractTokenTest.php

```diff
@@ -96,6 +97,19 @@ public function testSerialize()
         $this->assertEquals($token->getAttributes(), $uToken->getAttributes());
     }
 
+    public function testSerializeWithRoleObjects()
```

Comment:
Why not update `testSerialize`? Ie. [L91](https://github.com/symfony/symfony/pull/19778/files#diff-c8afeb16bbcfc3913e3eaee9dfcdda3bR91) => `$token = $this->getToken(array('ROLE_FOO', new Role('ROLE_BAR'));`